apps/frontend/src/renderer/styles/globals.css

```
@@ -154,6 +168,36 @@
   --error: #D84F68;
   --error-light: #FCE8EC;
 
+  /* Chart colors for data visualization */
+  --chart-1: #A5A66A;
+  --chart-2: #479FFA;
+  --chart-3: #4EBE96;
+  --chart-4: #D2D714;
+  --chart-5: #D84F68;
+
+  /* Font - serif fallback */
+  --font-serif: Georgia, Cambria, 'Times New Roman', Times, serif;
```

Stylelint (17.2.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)